docs/guides/using_TransformAnimationComponent.rst

```diff
@@ -0,0 +1,160 @@
+===================================================
+Animating Objects using TransformAnimationComponent
+===================================================
+
+TransformAnimationComponent can be used to easily animate scenes which have to be animated endlessly without relying on scripting for the same.
+
+It is quite easy to as it uses a keyframe based interface to achieve this.
+
+Setting up
+----------
+
+Create a new scene and assign the TransformComponent, ModelComponent and TransformAnimationComponent to it.
+
+Creating a new scene:
+
+.. figure:: images/AnimTestCreate.png
+
+Adding the Components by right clicking on the scene in inspector:
+
+.. figure:: images/AnimTestComp.png
+
+Select the EditorCamera:
+
+.. figure:: images/AnimTestCam.png
+
+Set the camera to EditorCamera and position it such that it has a good view of the scene.
+
+.. figure:: images/AnimTestView.png
+
+Select the EditorCamera as default camera for the scene.
+
+.. figure:: images/AnimTestDefCam.png
+
+Interface
+---------
+
+* Transition Type : How the transition must be between keyframes. Explaining the options:
```

Comment:
As requested in last review, `Explaining the options` seems like an awkward way to put it. Maybe replace it with something like `Following are the possible values for the Transition Type`